packages/pasteup/typography.ts

```diff
@@ -32,7 +32,7 @@ const fontScaleMapping: any = {
         9: { fontSize: 44, lineHeight: 48 },
     },
     body: {
-        1: { fontSize: 14, lineHeight: 20 },
+        1: { fontSize: 16, lineHeight: 24 },
```

Comment:
On reflection, I'm less in favour of changing `pasteup` for now. I know this value isn't being used anywhere else right now, but with this change there's so little difference between `body(1)` and `body(2)`, it seriously reduces its usefulness.

I would suggest sticking with `body(2)` for the `p` tag and overriding the `font-size` if we can't live with `17px`